lib/Sema/CSSimplify.cpp

```diff
@@ -10993,6 +10999,10 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
 
     return SolutionKind::Solved;
   }
+          
+  case FixKind::AllowWrappedValueMismatch: {
+      return recordFix(fix) ? SolutionKind::Error : SolutionKind::Solved;
```

Comment:
Nit: if there are going to be another changes - please adjust indent here to 2 spaces.